### PR TITLE
Add HipercooL source

### LINKED ISSUE #969: [Portuguese] Hipercool
[Hipercool](https://hiper.cool/)

### PR DESCRIPTION
**Pull request by alessandrojean**
Closes #969.

### CODE REVIEW

File: src/pt/hipercool/src/eu/kanade/tachiyomi/extension/pt/hipercool/Hipercool.kt [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        val result = response.asJsonArray()
+
+        if (result.size() == 0)
+            return MangasPage(emptyList(), false)
+
+        val latestMangas = result
+                .map { latestMangaItemParse(it.obj) }
+                .distinctBy { it.title }
+
+        return MangasPage(latestMangas, true)
```
Comment (arkon) [inline src/pt/hipercool/src/eu/kanade/tachiyomi/extension/pt/hipercool/Hipercool.kt]: Is there any way to actually tell if it's at the end?

File: src/pt/hipercool/src/eu/kanade/tachiyomi/extension/pt/hipercool/Hipercool.kt [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        val result = response.asJsonArray()
+
+        if (result.size() == 0)
+            return MangasPage(emptyList(), false)
+
+        val searchMangas = result
+                .map { searchMangaItemParse(it.obj) }
+                .distinctBy { it.title }
+
+        return MangasPage(searchMangas, true)
```
Comment (arkon) [inline src/pt/hipercool/src/eu/kanade/tachiyomi/extension/pt/hipercool/Hipercool.kt]: Same here.

**Comment by CarlosEsco** [comment]
you need to clean up the commit history.  it should only have files you are gonna change

**Comment by alessandrojean** [comment]
Yeap, I am trying to do that.

**Comment by alessandrojean** [comment]
@CarlosEsco, I don't know why Travis hasn't run the tests yet, but I think it's ready to merge now.

**Pull request merged**